### PR TITLE
{lib}[foss/2021b] ruffus v2.8.4 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/ruffus/ruffus-2.8.4-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+dependencies = [
+    ('Python', '3.9.6'),
+    ('Graphviz', '2.50.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+sanity_check_commands = [
+    "python -c 'import ruffus'",
```
Comment (boegel) [inline easybuild/easyconfigs/r/ruffus/ruffus-2.8.4-foss-2021b.eb:23]: This `import` is already done by default, so drop it and only do the import check on `ruffus.graph`:

```python
sanity_check_commands = ["python -c 'import ruffus.graph'"]

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/r/ruffus/ruffus-2.8.4-foss-2021b.eb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'ruffus'
+version = '2.8.4'
+
+homepage = 'http://www.ruffus.org.uk/'
+description = "Ruffus is a Computation Pipeline library for python. It is open-sourced, powerful and user-friendly, and widely used in science and bioinformatics."
```
Comment (boegel) [inline easybuild/easyconfigs/r/ruffus/ruffus-2.8.4-foss-2021b.eb:7]: You'll need to make this a multi-line string:
```python
description = """Ruffus is a Computation Pipeline library for python. It is open-sourced, powerful and
 user-friendly, and widely used in science and bioinformatics."""
```

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/r/ruffus/ruffus-2.8.4-foss-2021b.eb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+use_pip = True
+sanity_pip_check = True
+
+sanity_check_commands = [
+    "python -c 'import ruffus'",
+    "python -c 'import ruffus.graph'",
+]
+
+moduleclass = 'vis'
```
Comment (boegel) [inline easybuild/easyconfigs/r/ruffus/ruffus-2.8.4-foss-2021b.eb:27]: Maybe `lib` is more appropriate?

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by PetrKralCZ** [comment]
Test report by @PetrKralCZ
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3701.slaking.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/d44d17eebdd4d0889c6da1ba9003871b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3116.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/21ecadac56fe2cde35bf854bf06339e2 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**